### PR TITLE
[REVIEW] Set log_level when using LOGGING_LEVEL param

### LINKED ISSUE #953: add configurable log level
right now logging is impacting run time and theres no way to prevent that. We should wrap our logging statements in some kind of macro that lets us turn it on and off depending on levels set. Right now we only have something like that for flushing

### PR DESCRIPTION
**Pull request by Christian8491**
This PR uses the param `LOGGING_LEVEL` for set the level of logs we want. So the new name for the flush_level is `LOGGING_FLUSH_LEVEL`

Closes #953

### CODE REVIEW

File: engine/src/cython/initialize.cpp [hunk 1]
```diff
@@ -73,17 +78,48 @@ void create_logger(std::string fileName, std::string loggingName, int ralId, std
...
+	
+	// By default, will contains all sort of logs (more logs detail)
+	std::string logger_level_wanted = "trace";
+	auto log_level_it = config_options.find("LOGGING_LEVEL");
+	if (log_level_it != config_options.end()){
+		logger_level_wanted = config_options["LOGGING_LEVEL"];
+	}
+
+	// level of logs
+	if (logger_level_wanted == "off") {
```
Comment (williamBlazing) [inline engine/src/cython/initialize.cpp:94]: This mapping of string to spdlog enum is now happening twice can you please turn this into a function we can reuse.

File: engine/src/cython/initialize.cpp [hunk 2]
```diff
@@ -62,7 +62,12 @@ std::string get_ip(const std::string & iface_name = "eth0") {
 }
 
 // simple_log: true (no timestamp or log level)
-void create_logger(std::string fileName, std::string loggingName, int ralId, std::string flush_level, bool simple_log=true){
+void create_logger(std::string fileName,
+	std::string loggingName,
+	int ralId, std::string flush_level,
+	std::map<std::string, std::string> config_options,
```
Comment (williamBlazing) [inline engine/src/cython/initialize.cpp:68]: Here please pass in the logging level and not the config_options

File: engine/src/cython/initialize.cpp (hunk 1 above)
Comment (Christian8491) [reply engine/src/cython/initialize.cpp:94]: Currently `logger_level` is just once but the `flush_level` is mapping twice. So I am updating this `flush_level` in a separate function then we can reuse it.

File: engine/src/cython/initialize.cpp (hunk 2 above)
Comment (Christian8491) [reply engine/src/cython/initialize.cpp:68]: ok. Updated.

**Pull request merged**